### ISSUE TITLE
Terminal misses out on single forwarding events

### ISSUE THREAD

**Issue by Liongrass** [lightninglabs/lightning-terminal#740]
As my node forwards payments, they don't always show up in Terminal. It's hard for me to see any common characteristics, but once in a while I catch a forward appearing with `lncli fwdinghistory` but not in the Terminal UI. I have checked for errors in the browser output but haven't come across anything I found interesting or noteworthy.

**Actual behavior**

**To reproduce**

**System information**

Output from the `lncli` for an event that wasn't shown in Terminal:

```
        {
            "timestamp":  "1712029843",
            "chan_id_in":  "830313797997297665",
            "chan_id_out":  "812312593591500801",
            "amt_in":  "550725",
            "amt_out":  "550560",
            "fee":  "165",
            "fee_msat":  "165268",
            "amt_in_msat":  "550725268",
            "amt_out_msat":  "550560000",
            "timestamp_ns":  "1712029843362838795",
            "peer_alias_in":  "🧧laisee.org",
            "peer_alias_out":  "Bitrefill"
        }
```

**Comment by jamaljsr** [comment]
Thanks for the report @Liongrass. I have a suspicion as to what may be causing this, but I'm not 100% certain. 

Can you run `window.purgeTerminalData('txInfoState'); location.reload();` in your browser console? This will remove all of the forwards stored in the browser and re-fetch them from your LND node when you login. Once that completes, can you see if the missing forward appears?

**Comment by Liongrass** [comment]
Yes, it did indeed appear! Together with a few others. I feel a lot more successful now as a node operator...

**Comment by jamaljsr** [comment]
Ok great! Happy that it was a simple fix. I'll have to investigate how you got into that state in the first place. There must be a sub-second gap somewhere between when you last fetched forwards and when the forward arrived.
